app/bundles/CoreBundle/Form/Type/ConfigType.php

```diff
@@ -446,14 +446,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'choices' => [
                     'midnight'  => 'mautic.core.daterange.0days',
                     '-24 hours' => 'mautic.core.daterange.1days',
-                    '-1 week'   => $this->translator->transChoice('mautic.core.daterange.week', 1),
-                    '-2 weeks'  => $this->translator->transChoice('mautic.core.daterange.week', 2),
-                    '-3 weeks'  => $this->translator->transChoice('mautic.core.daterange.week', 3),
-                    '-1 month'  => $this->translator->transChoice('mautic.core.daterange.month', 1),
-                    '-2 months' => $this->translator->transChoice('mautic.core.daterange.month', 2),
-                    '-3 months' => $this->translator->transChoice('mautic.core.daterange.month', 3),
-                    '-1 year'   => $this->translator->transChoice('mautic.core.daterange.year', 1),
-                    '-2 years'  => $this->translator->transChoice('mautic.core.daterange.year', 2),
+                    '-1 week'   => $this->translator->transChoice('mautic.core.daterange.week', 1, ['%count%' => 1]),
+                    '-2 weeks'  => $this->translator->transChoice('mautic.core.daterange.week', 2, ['%count%' => 2]),
+                    '-3 weeks'  => $this->translator->transChoice('mautic.core.daterange.week', 3, ['%count%' => 3]),
+                    '-1 month'  => $this->translator->transChoice('mautic.core.daterange.month', 1, ['%count%' => 1]),
+                    '-2 months' => $this->translator->transChoice('mautic.core.daterange.month', 2, ['%count%' => 2]),
+                    '-3 months' => $this->translator->transChoice('mautic.core.daterange.month', 3, ['%count%' => 3]),
+                    '-1 year'   => $this->translator->transChoice('mautic.core.daterange.year', 1, ['%count%' => 1]),
+                    '-2 years'  => $this->translator->transChoice('mautic.core.daterange.year', 2, ['%count%' => 2]),
```

Comment:
It's strange that Symfony doesn't set the `%count%` var. It should based on the docs...

Aha, I linked docs to the new version. 2.8 must have it like this. Sorry for additional work, guys.